module-bsp/drivers/i2c/DriverI2C.hpp

```diff
@@ -49,6 +49,8 @@ namespace drivers
 
         virtual ssize_t Modify(const I2CAddress &addr, const uint32_t mask, bool setClr, const size_t size) = 0;
 
+        virtual int32_t GetLastError() = 0;
```

Comment:
I believe you missed adding simulator implementation? I can only see a `RT1051` one. In this case it won't compile for linux/simulator.

Reply:
Done